libs/blocks/brand-concierge/brand-concierge.js

```diff
@@ -37,6 +37,13 @@ function updateModalHeight() {
   modal.style.height = `${modalHeight}px`;
 }
 
+function updateInputHeight(el) {
+  const fieldInput = el.querySelector('.bc-input-field textarea');
+  if (!fieldInput) return;
+  fieldInput.style.height = 'auto';
+  fieldInput.style.height = `${fieldInput.scrollHeight}px`;
+}
```

Comment:
Maybe an unpopular opinion, but instead of constantly running some resize logic, why not let the browser handle it via a `contenteditable`? Something like [this](https://jsbin.com/rabosasaju/edit?html,css,output).